Twig/DoctrineExtension.php

```diff
@@ -37,9 +37,10 @@ class DoctrineExtension extends \Twig_Extension
     public function getFilters()
     {
         return array(
-            new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery')),
+            new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery'), array('deprecated' => true)),
             new \Twig_SimpleFilter('doctrine_pretty_query', 'SqlFormatter::format', array('is_safe' => array('html'))),
```

Comment:
well, this method does exactly the same than the new ``doctrine_highlight_sql``, except that one is using the profiler styles and the other one uses the raw SqlFormatter colors. This looks weird to me.